testing/server.test.js

```diff
@@ -0,0 +1,44 @@
+const request = require('supertest');
+const app = require('../server/index.js');
+const db = require('../database/index.js');
+
+// Check to see if a GET request is sent upon visiting root path
+describe('Test the root path', () => {
+  test('should response a GET method', () => {
+    return request(app).get('/').then((response) => { 
+      expect(response.statusCode).toBe(200);
+    });
+  });
+});
+
+describe('Test fetching room pics', () => {
+  test('should have the same unique exterior object', () => {
+    db.fetchRoomPics(12, (err, docs) => {
+      if (err) {
+        console.log('error during testing');
+        return;
+      }
+      const expected = {"id":69,"url":"https://s3-us-west-1.amazonaws.com/homesnap/exterior/h12.jpg","home_id":12,"order":1,"caption":"''"};
+      expect(JSON.stringify(docs[0])).toEqual(JSON.stringify(expected));
+    })
+  });
+});
+
+// Check to see if response exisits for visiting room page 
+// describe('Test response data type', () => {
+//   test('response should exist', () => {
+//     return request(app).get('/rooms/12').then((response) => {
+//       expect(response.data).toBeUndefined();
```

Comment:
Please let me know if I am reading this wrong, but is the reason you expect response.data to be undefined because your response object doesn't have a data property? 
If you are trying to see that a GET request returns the correct data, I would refactor this test so that you would expect() the response to equal whatever data you are expecting